exercises/practice/leap/.approaches/introduction.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+
+There are various ways to solve **leap**.
+Crystal has a leap method built in into the `Time` module (leap_year?).
+This exercise aims at practicing boolean logic thereby that method has been blocked.
+But for all other purposes when dealing with time should the built-in method be used.
+
+## General guidance
+
+The goal of this exercise is to use various boolean logic to respond to if a year is a leap year or not.
+
+## Approach: Using `divisible_by?`
+
+This is the most exemplar solution, and is the same way as the [`leap_year?`][leap-year] method is implemented (as of Crystal 1.11).
+`divisible_by?` offers a clear indication of what the method does and is easy to read.
+Compared to using `number % 4 == 0` which is harder to read and understand.
+Although `%` can get the remainder which `divisible_by?` can't.
```

Comment:
Neither `%` nor the concept of a  remainder was introduced. Perhaps we could add a single sentence here and a link to Wikipedia about modulo.